README.md

```diff
@@ -10,28 +10,186 @@ The features are available before an official release so that users and collabor
 
 ---
 
-TorchRL is an open-source Reinforcement Learning (RL) library for PyTorch. 
+**TorchRL** is an open-source Reinforcement Learning (RL) library for PyTorch. 
 
-It provides pytorch and python-first, low and high level abstractions for RL that are intended to be efficient, documented and properly tested. 
+It provides pytorch and **python-first**, low and high level abstractions for RL that are intended to be **efficient**, **modular**, **documented** and properly **tested**. 
```

Comment:
> On the documented and tested front it may be better to just show instead of tell, have a badge to show your test coverage and link to a high quality hosted documentation. As you build out more content and tutorials make sure to also link to them

Definitely
We already have a badge for the tests (passing or not), but not for coverage.
For the doc, thing is that right now the doc can't sit anywhere: readthedocs is not fb-approved, building a domaing just for the doc should be weighted against waiting for torchrl to migratet to github.com/pytorch: at this point the doc will be integrated in the general doc. Hence we're sitting on our hands for now. However, most functions and classes have an well documented docstring, so it's useful IMO to acknowledge that in the readme for now.